clearwater_config_manager/shared_config_plugin.py

```diff
@@ -68,5 +72,10 @@ def on_config_changed(self, value, alarm):
             safely_write(_file, value)
             run_command("/usr/share/clearwater/clearwater-queue-manager/scripts/modify_nodes_in_queue add apply_config")
 
+    def on_creating_etcd_key(self, value):
+        if value != _default_value:
+            _log.debug("Created non-default config key in etcd. Adding node to restart queue")
```

Comment:
Can you either remove this log, or add equivalent logs to the other plugins as well. I wonder if the log should also include the value (and parse down the log you've got).